sofiadinizms/vida-de-inseto
# Feature/level 5

### What I did

Created the Level Five general mechanics: get when the volume button is pressed, identifying the max/min volume
Added temporary assets to demonstrate said mechanics

### How to test

Git pull this branch
Run it on your iPhone
Check if the sound starts to play when the screen launches 
Use the volume up/down button to change de volume sound. 
See if the current volume will update every time you press the volume button. 
When the maximum volume is reached "volume máximo" is printed at debug area.
When the minimum volume is reached "volume mínimo" is printed at debug area.
When other values between 0.0 and 1.0 are reached "faz alguma coisa" is printed at debug area.

If you managed to successfully complete all the task described above, then everything is working correctly
